keeper-gradle-plugin/build.gradle.kts

```
@@ -48,6 +48,10 @@ tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach
     }
 }
 
+sourceSets {
+    findByName("test")?.resources?.srcDirs("$buildDir/pluginUnderTestMetadata")
```

nit: can we use `getByName` and fail fast if this isn't working?

---

daf340b